tests/sentry/api/endpoints/test_event_grouping_info.py

```diff
@@ -0,0 +1,94 @@
+from django.urls import reverse
+
+from sentry.event_manager import EventManager
+from sentry.testutils import APITestCase
+from sentry.testutils.silo import region_silo_test
+from sentry.types.issues import GroupType
+from sentry.utils import json
+from sentry.utils.performance_issues.performance_detection import (
+    EventPerformanceProblem,
+    PerformanceProblem,
+)
+from sentry.utils.samples import load_data
+
+
+@region_silo_test
+class ProjectOwnershipEndpointTestCase(APITestCase):
```

Comment:
Should this be named `EventGroupingInfoEndpointTestCase` or something like that?

Reply:
Whoops, great catch, thanks!